<launch>
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/kinect2/sd/image_color_rect" />
    </node>
</launch>